--- origsrc/Mem/AppRun	2005-01-23 06:43:55.000000000 -0600
+++ src/Mem/AppRun	2009-10-20 00:49:22.079708300 -0500
@@ -11,7 +11,7 @@ if [ "x$PLATFORM" = x ]; then
 	i?86) ARCH=ix86 ;;
   esac
 
-  PLATFORM=`uname -s`-$ARCH export PLATFORM
+  PLATFORM=CYGWIN-$ARCH ; export PLATFORM
 fi
 
 if [ "x$MAKE" = x ]; then
--- origsrc/Mem/AppletRun	2004-11-21 07:23:17.000000000 -0600
+++ src/Mem/AppletRun	2009-10-20 00:49:25.689713300 -0500
@@ -13,7 +13,7 @@ if [ "x$PLATFORM" = x ]; then
 	i?86) ARCH=ix86 ;;
   esac
 
-  PLATFORM=`uname -s`-$ARCH export PLATFORM
+  PLATFORM=CYGWIN-$ARCH ; export PLATFORM
 fi
 
 if [ "x$MAKE" = x ]; then
